config/linux/jenkins_dftimewolf_install.sh

```diff
@@ -30,7 +30,7 @@ sudo apt-get update -q
 sudo apt-get install -y ${PYTHON2_DEPENDENCIES}
 
 # Pending resolution of https://github.com/log2timeline/l2tdevtools/issues/233.
-sudo apt-get install -y python-pip
+sudo apt-get install -y python3-pip
```

Comment:
We also need to add `python3-setuptools` according to the recent jenkins build failures.